sdk/requests/__tests__/logout.test.js

```diff
@@ -0,0 +1,154 @@
+import logout from '../logout';
+import { getParameters } from '../../configuration';
+
+jest.mock('../../configuration');
+
+const mockAddEventListener = jest.fn();
+const mockLinkingOpenUrl = jest.fn(() => Promise.resolve());
+const missingParams = 'Missing required parameter(s): ';
```

Comment:
```suggestion
const missingParamsMessage = 'Missing required parameter(s): ';
```
Para no confundirnos con el objeto que tiene los parámetros. Si se acepta la sugerencia, es necesario cambiar la constante en todo el archivo.